src/Npgsql/NpgsqlDataSource.cs

```diff
@@ -227,6 +226,29 @@ public static NpgsqlDataSource Create(string connectionString)
     public static NpgsqlDataSource Create(NpgsqlConnectionStringBuilder connectionStringBuilder)
         => Create(connectionStringBuilder.ToString());
 
+    /// <summary>
+    /// Flushes the type cache for this data source.
+    /// Type changes will appear for connections only after they are re-opened from the pool.
+    /// </summary>
+    public void ReloadTypes()
+    {
+        using var connection = OpenConnection();
+        connection.ReloadTypes();
+    }
+
+    /// <summary>
+    /// Flushes the type cache for this data source.
+    /// Type changes will appear for connections only after they are re-opened from the pool.
+    /// </summary>
+    public async Task ReloadTypesAsync()
```

Comment:
This is missing a CancellationToken parameter; I can see that the NpgsqlConnection.ReloadTypesAsync() overload is similarly missing it, I'm not sure why - maybe add it there as well as part of this PR?

Reply:
Done